compiler/src/dotty/tools/dotc/reporting/MessageKind.scala

```diff
@@ -37,5 +38,6 @@ enum MessageKind:
       case PatternMatchExhaustivity => "Pattern Match Exhaustivity"
       case MatchCaseUnreachable => "Match case Unreachable"
       case PotentialIssue => "Potential Issue"
+      case UnusedSymbol => "Unused symbol"
```

Comment:
If you could change this to "Unused Symbol" as it seems we define them in Pascal case, you can also take the chance to change Match Case Unreachable. Tests will fail after this change.